### ISSUE TITLE
Return code 1 after upgrade to 2.1.2

### ISSUE THREAD

**Issue by pascalberger** [carsdotcom/gulp-sonar#19]
After the release of version 2.1.2 we get the following error in our build scripts:

```
<Buffer 27 73 68 27 20 69 73 20 6e 6f 74 20 72 65 63 6f 67 6e 69 7a 65 64 20 61 73 20 61 6e 20 69 6e 74 65 72 6e 61 6c 20 6f 72 20 65 78 74 65 72 6e 61 6c 20 ... >
Return code: 1.
D:\work\86\s\node_modules\gulp-sonar\index.js:84
             throw new PluginError('gulp-sonar', format('Return code: %d.', code));
             ^
Error: Return code: 1.
```

Works fine with version 2.1.0. But fails with version 2.1.1 / 2.1.2. Sonar version is 5.6, JavaScript plugin 2.14 (Analysis is for a JS file).

**Comment by machellerogden** [comment]
I'm looking into it today. A patch will be published.

**Comment by machellerogden** [comment]
We've fixed the issue by rolling forward (reverting) to what essentially is the v2.1.0 release. There was a snafu with the publishing so we landed on v2.1.5. This means that v2.1.5 is essentially v2.1.0 again. Sorry for the confusion.
